src/test/common/test_mutex.cc

```diff
@@ -0,0 +1,66 @@
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
+// vim: ts=8 sw=2 &smarttab
+/*
+ * Ceph - scalable distributed file system
+ *
+ */
+
+#include <common/Mutex.h>
+#include "gtest/gtest.h"
```

Comment:
@tchaikov: The unittest_mutex_debug is already a part of src/test/common/CMakeLists.txt, lines 191-194. Can you confirm?